src/IniFile.h

```diff
@@ -61,6 +61,16 @@ class cIniFile:
 	/** Removes the UTF-8 BOMs (Byte order makers), if present. */
 	void RemoveBom(AString & a_line) const;
 
+public:
+
+	/** Reads the contents of the specified string
+	Returns true if successful, false otherwise. */
+	bool ReadString(const AString & a_Data);
+
+	/** Writes data stored in class to the specified string.
+	Returns true on success, false on failure. */
+	bool WriteString(AString & a_Data) const;
+
 	// tolua_begin
```

Comment:
C++ classes can have _two_ public sections in the same header file? Well it seems so at least, but this would better be put under the extant section just below. 

And these two functions seem like they could be useful exported to the API as well, so no problem with them being inside the `tolua_begin` section.

Reply:
I could not make the tests pass because of an error about documentation which was non-existent. So, in a first step I added the comments you can see, but it did not work. I was unable to understand what the test tool wanted, but I understood it was "because" of the lua API. I finally moved the declarations to not be bothered by this, since adding functions to the lua API is not my concern.

And yes, C++ can have their "sections" declared several times. I use it a lot to separate `private data` from `private functions` from `private constants`, etc... Same for `public members` and `public constructors/destructors` and `public virtual members`...
But these are not real "sections", these are "access modifiers", exactly like in Java for example, except that it is applied to several lines instead of only one in Java.

Reply:
Ah, this is just an organisational pattern that I had not seen before, I should really try to use the technical terms, of course I do know the function of the access modifiers :)

The documentation for lua needs to be added to the Server/Plugins/APIDump/APIDesc.lua file